jetty-util/src/main/java/org/eclipse/jetty/util/thread/ExecutorThreadPool.java

```diff
@@ -302,7 +302,7 @@ protected void doStop() throws Exception
     @Override
     public void join() throws InterruptedException
     {
-        _executor.awaitTermination(getStopTimeout(), TimeUnit.MILLISECONDS);
+        _executor.awaitTermination(Long.MAX_VALUE, TimeUnit.MILLISECONDS);
```

Comment:
Why are we even doing `awaitTermination()`?

Isn't there a better choice?

Reply:
I guess a 292 million year wait isn't going to cause anybody any concerns.

Reply:
Doesn't look like there is any better way to do it other than `awaitTermination()`.

It doesn't look like waiting only `stopTimeout` caused many concerns either, it has been like that for over 2 years and only one issue filed against it. I guess it just wasn't used much.